nectarchain/calibration/NectarGAIN.py

```diff
@@ -255,41 +271,66 @@ def NG_LikelihoodSignal_Chi2(self,pp,res,mu2,n,muped,sigped,lum,charge,nEvents):
     
+# To call NG_LikelihoodSignal_Chi2 wit hthe data loaded, not all free parameters (pp and n are fixed (as they are independent from the gain, should be the same for all pixels althogh unchecked), will be usefull for the combined fit with the 1000V
+# Paramters fixed after Sami did a avergae on 50 pixels just have an idea and gain time
     def Chi2SignalFixedModel(self,res,mu2,muped,sigped,lum):
         return self.NG_LikelihoodSignal_Chi2(self.pp,res,mu2,self.n,muped,sigped,lum,self.chargeSignal,self.histoSignal)
     
+# Chi2 (used now) for the ped at 1000V (alwasy work)
     def Chi2Ped(self,muped,sigped):
         return self.NG_LikelihoodPedestal_Chi2(muped,sigped,self.chargePed,self.histoPed)
     
+# Chi2 (used now) for the signal (HHV: high high voltage ie 1400V) at 1400V
+# Ideal/final for 1400V
     def Chi2SignalHHV(self,pp,res,mu2,n,muped,sigped,lum):
         return self.NG_LikelihoodSignal_Chi2(pp,res,mu2,n,muped,sigped,lum,self.chargeSignalHHV,self.histoSignalHHV)
     
+# Chi2 (used now) for the ped at 1400V (always work)
+# Ideal for file with only pedestal/noise (not for us now))
     def Chi2PedHHV(self,muped,sigped):
         return self.NG_LikelihoodPedestal_Chi2(muped,sigped,self.chargePedHHV,self.histoPedHHV)
         
+# Chi2 (not tested yet) for the combined pedestal + signal (at 1000V)
+# If pp and n are free it will not converge, requires a pedestal run like above, ideal for 1000V file + noise/pedestal file
+# Not for now
     def Chi2CombiSignalAndPed(self,pp,res,mu2,n,muped,sigped,lum):
-        return self.Chi2Signal(pp,res,mu2,n,muped,sigped,lum)+self.Chi2Ped(muped,sigped)
+        return self.Chi2Signal(self.pp,res,mu2,self.n,muped,sigped,lum)+self.Chi2Ped(muped,sigped)
+        #return self.Chi2Signal(pp,res,mu2,n,muped,sigped,lum)+self.Chi2Ped(muped,sigped)
         
+# Chi2 (not tested yet) for the combined pedestal + signal (at 1400V)
+# Requires a pedestal file so not for now
     def Chi2CombiSignalAndPedHHV(self,pp,res,mu2,n,muped,sigped,lum):
         return self.Chi2SignalHHV(pp,res,mu2,n,muped,sigped,lum)+self.Chi2PedHHV(muped,sigped)
         
+# Chi2 (not tested yet) for the combined pedestal + signal (1000V) + signal (1400V)
+# Requires a pedestal file so not for now
+# Ideal/final for ped + 1000V + 1400V
     def Chi2AllCombined(self,pp,res,mu2,mu2HHV,n,muped,mupedHHV,sigped,lum,lumHHV):
         return self.Chi2CombiSignalAndPed(pp,res,mu2,n,muped,sigped,lum)+self.Chi2CombiSignalAndPedHHV(pp,res,mu2HHV,n,mupedHHV,sigped,lum)
         
+# Chi2 (not tested yet) for the combined signal (1000V) + signal (1400V)
+# Ideal/final for 1000V + 1400V
+    def Chi2AllNoPedCombined(self,pp,res,mu2,mu2HHV,n,muped,mupedHHV,sigped,lum,lumHHV):
+        return self.Chi2Signal(pp,res,mu2,n,muped,sigped,lum)+self.Chi2SignalHHV(pp,res,mu2HHV,n,mupedHHV,sigped,lum)
+        
     ####### Compute Start Parameters ######
-    
+   
+# "Smart" values to start the fit 
```

Comment:
Are the changed parameters in this method relaxed to ease the fit ?

Reply:
OK, it was not clear why some values changed, whether this was deemed robust, or still work in progress. All good for now, then.